requirements.txt

```diff
@@ -23,6 +23,7 @@ pandas>=1.1.0,<2.0.0; python_version <= "3.8"
 pandas>=1.1.3,<2.0.0; python_version == "3.9"
 pandas>=1.3.0,<2.0.0; python_version >= "3.10"
 # patch version updates `typing_extensions` to the needed version
+pika==1.3.1
```

Comment:
There are 3 steps here.
1. Remove this from requirements.txt
2. Create a new requirements file under `/reqs` called `requirements-dev-agent.txt`
    * https://github.com/great-expectations/great_expectations/tree/develop/reqs
3. Add this `requirements-dev-agent.txt` file this [requirements-types.txt](https://github.com/great-expectations/great_expectations/blob/develop/requirements-types.txt)
```suggestion
```

This will make it an optional dependency and `pika` and any other dependencies installable as "extra" dependencies.
```
pip install 'great_expectations[agent]'
```

Reply:
Updated the suggestion to call our extra dependencies `cloud` instead of `agent`, this is based on some discussion from Monday's standup.

cc @DrewHoo

Reply:
added in [6928465](https://github.com/great-expectations/great_expectations/pull/7837/commits/6928465da4e2e63309254d6b0b32598de9f4b411)